### ISSUE TITLE
v1.4 Relations.intersect() doesn't allow Polygon argument

### ISSUE THREAD

**Issue by jstemo-titan** [alexbol99/flatten-js#149]
Thank you for this library.

First saw this in 1.4. Also appears in 1.4.1

```
const polygon = new Flatten.Polygon(new Flatten.Box(2, 2, 3, 3));
const box = new Flatten.Box(1, 1, 10, 10);
Relations.intersect(polygon, box);
```

Argument of type 'Polygon' is not assignable to parameter of type 'Shape'.
  Property 'scale' is missing in type 'Polygon' but required in type 'Shape'.

**Comment by alexbol99** [comment]
Hi, @jstemo-titan 
This is a typescript bug, correct?
I submitted a fix. I will release it soon

**Comment by jstemo-titan** [comment]
Hi Alex,

Yes this is a typescript bug – sorry for not being more clear.
Thank you for looking into this.

-Jeremy

From: Alex Bol ***@***.***>
Date: Tuesday, August 29, 2023 at 8:00 AM
To: alexbol99/flatten-js ***@***.***>
Cc: Stemo, Jeremy ***@***.***>, Mention ***@***.***>
Subject: Re: [alexbol99/flatten-js] v1.4 Relations.intersect() doesn't allow Polygon argument (Issue #149)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Hi, @jstemo-titan<https://github.com/jstemo-titan>
This is a typescript bug, correct?
I submitted a fix. I will release it soon

**Comment by alexbol99** [comment]
Fixed in the patch v1.4.3
